src/app/controllers/frontend.server.controller.ts

```diff
@@ -1,16 +1,24 @@
-'use strict'
+import ejs from 'ejs'
```

Comment:
this file should be moved into a `frontend` module

Reply:
shifted